app/assets/javascripts/live_search.js

```diff
@@ -115,6 +115,14 @@
       this.$suggestionsBlock.setAttribute('data-search-query', this.currentKeywords())
     }
     if (GOVUK.Ecommerce) { GOVUK.Ecommerce.start() }
+
+    if (GOVUK.Modules) {
+      var searchResults = document.querySelectorAll("[data-module='gem-track-click ga4-enhanced-ecommerce-tracker']")
```

Comment:
Is it possible to only search here for `document.querySelectorAll("[data-module='ga4-enhanced-ecommerce-tracker']")`? The reason I ask is that `gem-track-click` is from the existing analytics code, and will disappear or change eventually, at which point this will stop working.

Reply:
Just to document/reflect the discussion we had yesterday - the ecommerce tracking code will no longer be structured as a module so we won't be using this approach any more.